src/std/10-std-builder.md

```diff
@@ -1 +1,175 @@
 # Builder
+
+The **builder** module defines a `builder` type that allows to build and install C++ modules to be used with Feral
+
+A guide on how to write and build modules for Feral can be read [here](https://levelup.gitconnected.com/writing-c-modules-for-feral-391c30ac7739).
```

Comment:
please use this URL:
https://medium.com/p/writing-c-modules-for-feral-391c30ac7739?source=email-852839018f8a--writer.postDistributed&sk=b2de39e5f849e75e3b4ea9bdeeb1b4db

it does not require medium membership